public/css/style.css

```diff
@@ -0,0 +1,188 @@
+*{
+  margin: 0 auto;
```

Comment:
you should make separate  file to (header.css , footer.css) 
because you separate them in partials